Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorState.cs

```diff
@@ -113,16 +113,6 @@ internal struct InputActionsEditorState
             m_CutElements = cutElements;
         }
 
-        private static int AdjustSelection(SerializedObject serializedObject, string propertyName, int index)
```

Comment:
This AdjustSelection was dead code introduced in #1850 I decided to remove while merging this with develop.